### PR TITLE
Added Terraform Resource

### PR DESCRIPTION
**Pull request by JAG-010**
Hi,

I have added Terraform resources to this wonderful repo

PS: First time participating on Hacktoberfest 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -520,6 +521,17 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Terraform 
+
+- [Terraform Study Guide](https://github.com/ari-hacks/terraform-study-guide)
+- [Zeal Vora - Udemy course](https://www.udemy.com/course/terraform-beginner-to-advanced/)
```
Comment (shahednasser) [inline README.md:527]: Replace with the title of the course

File: README.md [hunk 2]
```diff
@@ -520,6 +521,17 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [Deep Dive - Terraform (Pluralsight)](https://www.pluralsight.com/courses/deep-dive-terraform)
+- [freecodecamp YT video](https://youtu.be/SLB_c_ayRMo)
```
Comment (shahednasser) [inline README.md:529]: Replace with the title of the video

File: README.md [hunk 3]
```diff
@@ -520,6 +521,17 @@ To add a resource or contribute, please check out our contribution guide [here](
...
+- [KodeKloud YT Video](https://youtu.be/YcJ9IeukJL8)
```
Comment (shahednasser) [inline README.md:530]: Replace with the title of the video

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @JAG-010 for your contribution and I hope it wasn't so hard!

I just have some simple comments below I hope you can fix them before I merge this PR.

File: README.md (hunk 1 above)
Comment (JAG-010) [reply README.md]: Thanks @shahednasser, I have updated with title of the course

File: README.md (hunk 2 above)
Comment (JAG-010) [reply README.md]: I have updated with title of the course

File: README.md (hunk 3 above)
Comment (JAG-010) [reply README.md]: I have updated with title of the course

**Review by JAG-010 — COMMENTED** [review]
I have updated with title of the course

**Comment by shahednasser** [comment]
Awesome thanks!

**Pull request merged**